Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Dark Crystal

## Changelog:
- Added wc_historical_artifacts events to create and add the dark_crystal

## Developer changelog:
- Added wc_historical_artifacts.0200 and .0201
- Added wc_historical_artifacts.0200 to on_combat_actions
- .0200 triggers on victory against an orcish fel army with caster_unit MAA
- .0201 creates the dark_crystal artifact and asks the player what to do with it

## Tests:
- Defeat a corruptist army that has caster MAAs
- Make sure the player gets the dark crystal
- Make sure the "mysterious" character joins your court
- Search for throw cases where the player isn't holy_light

# How to test: Start as Stormwind in the first bookmark starting date and defeat a corruptist army with an MAA. "event wc_historical_artifacts.0201" in the console command triggers the event quicker testing.
